mythril/src/vm.rs

```diff
@@ -24,6 +24,9 @@ use spin::RwLock;
 
 static BIOS_BLOB: &'static [u8] = include_bytes!("blob/bios.bin");
 
+//TODO(alschwalm): this should always be reported by the relevant MSR
+const GUEST_LOCAL_APIC_ADDR: GuestPhysAddr = GuestPhysAddr::new(0xfee00000);
```

Comment:
Can you use this in the MADT we pass to the guest [here](https://github.com/mythril-hypervisor/mythril/blob/master/mythril/src/kmain.rs#L66)

Reply:
Done